lib/cdo/mailjet.rb

```diff
@@ -108,6 +105,23 @@ def self.delete_contact(email)
     end
   end
 
+  def self.add_to_contact_list(contact, list_id)
+    return unless enabled?
+    return if contact.nil?
+
+    Mailjet.configure do |config|
```

Comment:
Could this somehow be abstracted? Or will we sometimes be using different keys/secrets/versions?

Reply:
1. It's different between contacts and emails 🫠 But I think it could likely be abstracted somewhat, let me play with it

Reply:
omg the API version 3 vs 3.1 🤦🏻‍♀️